### ISSUE TITLE
Crash when exiting character profiles

### ISSUE THREAD

**Issue by Graslu** [fgsfdsfgs/perfect_dark#124]
Sometimes, usually the first time booting up the game after a new update, if you go to the Character Profiles in CI and then exit by pressing E twice it'll crash without any error message.

**Comment by fgsfdsfgs** [comment]
Can't get that to happen. Does it still happen after the latest update?

**Comment by Graslu** [comment]
It happened with that last one yes, and another from last night. It seems to happen only the first time you launch the game after a fresh Windows boot.

**Comment by mmillar-bolis** [comment]
Windows 10 user here; I can't reproduce the issue from a fresh boot either. Do you have anything that hooks to the process, such as Steam Overlay, Rivatuner, or the like?

**Comment by Graslu** [comment]
I do not, no, but I am on Windows 11. I'll try to get a video tomorrow if I can get it to happen again. It did happen at least 4 times since last night.

**Comment by fgsfdsfgs** [comment]
I ought to add a crash handler to the game.

**Comment by Graslu** [comment]
I can't seem to replicate it again, after a fresh boot and installation of the latest update... Strange, haven't changed anything on my end and it seemed fairly consistent on my end.

**Comment by fgsfdsfgs** [comment]
There is now a crash handler. If you get a crash, make sure to post the screenshot of the error message.

**Comment by djdv** [comment]
If you're building from source and were updating an existing build, I have an anecdote that may be related.

There was at least one occasion where I pulled the latest commit, called `make` and ran `pd` but it would exit consistently (I forget what triggered it).
I deleted the build artifacts and rebuilt from a clean source tree. Afterwards I restored the ini and eeprom files and had no issues even though it was the same commit, settings, save, etc.

My assumption is that maybe some of the necessary objects were not rebuilt when trying to update and that may have been causing a crash somewhere. But I'm not certain about it.

Extra:
It's also possible I mixed optimization flags between those builds, using `O3` for build one and `O0` (the default at the time) for build two.
Since the default compiler flags in the makefile changed to `Og`, I wonder if this is also somehow related.
